CHANGELOG.md

```diff
@@ -1,7 +1,3 @@
-Must provide GITHUB_AUTH
```

Comment:
@tylerturdenpants not sure what was going on here, the changelog doesn't seem to have been updating properly with `release-it`